### DISCUSSION TITLE
Upgrade `react` version of IDPT

### DISCUSSION THREAD

**Discussion by sureshHARDIYA** [sureshHARDIYA/idpt#98] [Q&A] [answered]
It is time that this project deserves some `love`. What would be best approach to upgrade react-version of this project? 

Should we try to break this repo into BE and FE repo? Will that help for maintainability?

**Comment by anjumukhiya** [comment]
That is good initiative, we should do that.

**Comment by sureshHARDIYA** [comment, marked as answer]
1. Upgrade `react`
2. Upgrade `react-router-dom`
